specification/security/resource-manager/Microsoft.Security/stable/2019-08-01/iotSecuritySolutions.json

```diff
@@ -0,0 +1,659 @@
+    "TagsResource": {
+      "properties": {
+        "tags": {
+          "type": "object",
+          "additionalProperties": {
+            "type": "string"
+          },
+          "description": "Resource tags"
+        }
+      },
+      "description": "A container holding only the Tags for a resource, allowing the user to update the tags."
+    },
+    "IoTSecuritySolutionsList": {
+      "description": "List of iot solutions",
+      "required": [
+        "value"
+      ],
+      "properties": {
+        "value": {
+          "type": "array",
+          "description": "List of security solutions",
+          "items": {
+            "$ref": "#/definitions/IoTSecuritySolutionModel"
+          }
+        },
+        "nextLink": {
+          "readOnly": true,
+          "type": "string",
+          "description": "The URI to fetch the next page."
+        }
+      }
+    },
+    "IoTSecuritySolutionModel": {
+      "type": "object",
+      "description": "Security Solution",
+      "properties": {
+        "location": {
+          "type": "string",
+          "description": "The resource location."
+        },
+        "properties": {
+          "x-ms-client-flatten": true,
+          "description": "Security Solution data",
+          "$ref": "#/definitions/IoTSecuritySolutionProperties"
+        }
+      },
+      "allOf": [
+        {
+          "$ref": "../../../common/v1/types.json#/definitions/Resource"
+        },
+        {
+          "$ref": "#/definitions/TagsResource"
+        }
+      ]
+    },
+    "IoTSecuritySolutionProperties": {
+      "type": "object",
+      "description": "Security Solution setting data",
+      "properties": {
+        "workspace": {
+          "type": "string",
+          "description": "Workspace resource ID"
+        },
+        "displayName": {
+          "type": "string",
+          "description": "Resource display name."
+        },
+        "status": {
+          "type": "string",
+          "enum": [
+            "Enabled",
+            "Disabled"
+          ],
+          "default": "Enabled",
+          "description": "Security solution status",
+          "x-ms-enum": {
+            "name": "SecuritySolutionStatus",
+            "modelAsString": true
+          }
+        },
+        "export": {
+          "type": "array",
+          "items": {
+            "enum": [
+              "RawEvents"
+            ],
+            "type": "string",
+            "x-ms-enum": {
+              "name": "ExportData",
+              "modelAsString": true,
+              "values": [
+                {
+                  "value": "RawEvents",
+                  "description": "Agent raw events"
+                }
+              ]
+            }
+          },
+          "description": "List of additional export to workspace data options"
+        },
+        "disabledDataSources": {
+          "type": "array",
+          "items": {
+            "enum": [
+              "TwinData"
+            ],
```

Comment:
What's the intent behind this single-value enum?

Reply:
Because we were recommended to define it as enum instead of boolean in order to be able to expand it

Reply:
Ok. Sounds good.

---
In reply to: [308688131](https://github.com/Azure/azure-rest-api-specs/pull/6766#discussion_r308688131) [](ancestors = 308688131)